README.md

```diff
@@ -6,3 +6,188 @@ Distributed service to interact with development boards.
 * [boardswarm-protocol](boardswarm-protocol/README.md) - Boardswarm protocol definitions
 * [boardswarm-cli](boardswarm-cli/README.md) - Boardswarm command line client
 * [boardswarm-client](boardswarm-client/README.md) - Boardswarm client library
+
+## Installation
+
+These instructions are for configuring a simple unencrypted setup with static
+JWKS based authentication.
+
+### Building
+
+Boardswarm needs rustc version 1.70 or newer, this rules out a number of stable
+distributions for now (the kind of thing you may want to run your
+infrastructure on), however if needed we can build on a system that provides
+the required level of rust support and, as rust builds static binaries, we can
+copy them to the target server once built. As boardswarm is also quite new and
+undergoing active development, we'll build the latest from git.
+
+1. Install required tools/libraries to build boardswarm on a suitable system.
+   As we're pulling the latest from git, we'll need that installed too.
+   Assuming a Debian based system:
+   ```
+   $ sudo apt install rustc git protobuf-compiler libudev-dev
+   ```
+1. Clone the boardswarm repository:
+   ```
+   $ git clone https://github.com/boardswarm/boardswarm.git
+   ```
+1. We can now build boardswarm:
+   ```
+   $ cd boardswarm
+   $ cargo build
```

Comment:
```suggestion
   $ cargo build --release
```
for normal usage you always want to build a release binary otherwise it's rather unoptimised

Reply:
Added